core/templates/components/state-editor/state-editor-properties-services/state-editor.service.ts

```diff
@@ -201,15 +204,17 @@ export class StateEditorService {
     return cloneDeep(this.interaction);
   }
 
+  // Function will return null if interactionId does not exist.
   getAnswerChoices(
       interactionId: string,
-      customizationArgs: InteractionCustomizationArgs): AnswerChoice[] {
+      customizationArgs: InteractionCustomizationArgs): AnswerChoice[] | null {
     if (!interactionId) {
       return null;
     }
     // Special cases for multiple choice input and image click input.
     if (interactionId === 'MultipleChoiceInput') {
-      return (<MultipleChoiceInputCustomizationArgs> customizationArgs)
+      return <AnswerChoice[]>(
+        <MultipleChoiceInputCustomizationArgs> customizationArgs)
```

Comment:
break the line before the `)` and then have `).choices…` on the next line.

Reply:
done